lib/purserl.ex

```diff
@@ -10,10 +10,8 @@ defmodule DevHelpers.Purserl do
 
   ###
 
-  @name :purserl_compiler
-
   def start(config) do
-    case GenServer.start(__MODULE__, config, name: @name) do
```

Comment:
I hope you realise that this is a massive breaking change, and that the mix compiler config will have to be updated in all projects that use this tool. Do you really need to rename it?

Reply:
Nvm, it's named `:purserl` externally still